Content.Shared/_DV/Traits/Assorted/UncloneableComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Traits.Assorted;
+
+/// <summary>
+/// This is used for the unrevivable trait as well as generally preventing revival.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class UncloneableComponent : Component
+{
+    /// <summary>
+    /// A field to define if we should display the "Genetic incompatibility" warning on health analysers
+    /// </summary>
+    [DataField]
+    public bool Analyzable = true;
+
+    /// <summary>
+    /// Can this player be cloned using a cloning pod?
+    /// </summary>
+    [DataField]
+    public bool Cloneable = false;
```

Comment:
Why add this field?
The entire component marks an entity as uncloneable, this should never be true and can just be assumed to be false